tests/foreman/api/test_registration.py

```diff
@@ -365,3 +365,83 @@ def test_negative_verify_bash_exit_status_failing_host_registration(
     # verify status code when registrationCommand fails to register on host
     assert result.status == 1
     assert 'Couldn\'t find activation key' in result.stderr
+
+
+@pytest.mark.parametrize('job_type', ['ansible', 'ssh'])
+@pytest.mark.rhel_ver_match('8')
```

Comment:
```suggestion
@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

Reply:
In my opinion, we should test on all rhel versions. wdyt @Gauravtalreja1 @shubhamsg199 ?

Reply:
I believe from functionality perspective it would be same for all rhel versions